Resources/Prototypes/_Moffstation/Entities/Mobs/Customization/Markings/undergarments.yml

```
@@ -0,0 +1,90 @@
+- type: marking
+  id: UndergarmentBottomBoxersAvali
+  bodyPart: UndergarmentBottom
+  markingCategory: UndergarmentBottom
+  speciesRestriction: [Avali]
+  coloring:
+    default:
+      type: null
+      fallbackColor: '#FFFFFF'
```

```suggestion
  coloring:
    default:
      type: !type:SimpleColoring
```
The fallback is used if the `default.type` is null, so set that one and the fallback isn't necessary.

If this fails to deserialize, change it to
```csharp
  coloring:
    default:
      type: !type:SimpleColoring
        color: "#FFFFFF"
```
. The default value of `SimpleColoring`'s `color` is that same white, but possibly the serializer won't like an empty object with just a type tag, I'm not sure.